guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+{Project} supports importing images from local and external registries.
+{Project} itself can act as an image registry for hosts.
+However, hosts cannot push changes back to the registry.
+
+[[Repository]]
+Repository:: Provides storage for a collection of content.
+
+[[Resource_Type]]
+Resource Type:: Refers to a part of {Project} infrastructure, for example host, {SmartProxy}, or architecture.
+Used in permission filtering.
+
+[[Role]]
+Role:: Specifies a collection of permissions that are applied to a set of resources, such as hosts.
+Roles can be assigned to users and user groups.
+{Project} provides a number of predefined roles.
+
+[[SCAP_content]]
+SCAP content:: A file containing the configuration and security baseline against which hosts are checked.
+Used in compliance policies.
+
+[[Scenario]]
+Scenario:: A set of predefined settings for the {Project} CLI installer.
+Scenario defines the type of installation, for example to install {SmartProxyServer} execute `{installer-scenario-smartproxy}`.
+Every scenario has its own answer file to store the scenario settings.
+
+ifndef::orcharhino,satellite[]
+[[{smart-proxy-context}]]
+{SmartProxy}:: A {SmartProxy} component that can integrate with external services, such as *DNS* or *DHCP*.
+{SmartProxies} are useful for {Project} deployment across various geographical locations.
+endif::[]
+
+[[Standard_Operating_Environment_SOE]]
+Standard Operating Environment (SOE):: A controlled version of the operating system on which applications are deployed.
+
+[[Subnet_Image]]
+Subnet Image:: A type of generic image for PXE-less provisioning that communicates through {SmartProxyServer}.
+
+[[Subscription]]
+Subscription:: An entitlement for receiving content and service from Red{nbsp}Hat.
+
+[[Synchronization]]
+Synchronization:: Refers to mirroring content from external resources into the {ProjectName} Library.
+
+[[Sync_Plan]]
+Sync Plan:: Provides scheduled execution of content synchronization.
+
+[[Task]]
+Task:: A background process executed on the {Project} or {SmartProxyServer}, such as repository synchronization or content view publishing.
+You can monitor the task status in the {ProjectWebUI} under *Monitor* > *{Project} Tasks* > *Tasks*.
+
+ifndef::orcharhino,satellite[]
+[[Trend]]
+Trend:: A means of tracking changes in specific parts of {Project} infrastructure.
+Configure trends in {ProjectWebUI} under *Monitor* > *Trends*.
+Requires foreman_statistics plugin on your {ProjectServer}.
+endif::[]
+
+[[Updating]]
+Updating {Project}:: The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a z-stream release to the next, for example {Project} {ProjectVersion}.0 to {Project} {ProjectVersion}.1.
+
+[[Upgrading]]
+Upgrading {Project}:: The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
+
```

Comment:
As a user, I would look for these definitions in the docs for updating and upgrading, not here.

Reply:
I think those terms provide value.